src/main.cpp

```diff
@@ -2,6 +2,7 @@
  * Include the Geode headers.
  */
 #include <Geode/Geode.hpp>
+#include <Geode/modify/MenuLayer.hpp>
```

Comment:
no

Reply:
why do you want bad looking code

Reply:
it's not about code looking bad, it's about readability. geode's standard is to include the modify classes right before modify is included, since the inclusion and the modify itself is closely related to each other.

Reply:
> it's not about code looking bad, it's about readability. geode's standard is to include the modify classes right before modify is used, since the inclusion and the modify itself is closely related to each other.

@altalk23 fixed.